src/spryker/harness.yml

```diff
@@ -28,6 +28,8 @@ attributes:
     mode: development
   php:
     version: 7.2
+    ini:
+      opcache.max_accelerated_files: 32531
```

Comment:
Is this the number of files this spryker release contains? could we set it to a bit higher number like 40k?

Reply:
Only 28867 files in the spryker install when finished with dev dependencies and 22570 without.
Value will be rounded up to nearest prime number in the list from https://www.php.net/manual/en/opcache.configuration.php#ini.opcache.max-accelerated-files
Next prime number after that is 65407

Reply:
oh, didn't know that 👍

Reply:
32531 is one of the prime numbers though, should we increase to 65407 to account for project files/extra deps?

Reply:
I have more than 35k php files in total in teufel project, so yeah we should increase this number